dask/dataframe/tests/test_rolling.py

```diff
@@ -205,3 +205,32 @@ def test_rolling_repr():
                                     'Rolling [center=False,window=4,axis=0]',
                                     'Rolling [axis=0,window=4,center=False]',
                                     'Rolling [axis=0,center=False,window=4]'}
+
+
+class TestRollingTimeWindow(object):
```

Comment:
FWIW this will be (I think) the first testing class within the codebase.  Historically we've used bare functions and pytest fixtures.